Runtime/Tx/RaycastLiDARTx.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using UnitySensors.Sensor.LiDAR;
+using sensor_msgs.msg;
+using Unity.Collections;
+using Unity.Jobs;
+using UnityEngine;
+using UnitySensors.Data.PointCloud;
+
+
+namespace ProBridge.Tx.Sensor
+{
+    [RequireComponent(typeof(RaycastLiDARSensor))]
+    public class RaycastLiDARTx : ProBridgeTxStamped<PointCloud2>
+    {
+        private RaycastLiDARSensor sensor;
+        private IPointsToPointCloud2MsgJob<PointXYZI> _pointsToPointCloud2MsgJob;
+        private JobHandle _jobHandle;
+        private NativeArray<byte> tempData;
+
+
+        protected override void OnStart()
+        {
+            sensor = GetComponent<RaycastLiDARSensor>();
+            
+            
+            data.fields = new PointField[3];
+            for (int i = 0; i < 3; i++)
+            {
+                data.fields[i] = new PointField();
+                data.fields[i].name = ((char)('x' + i)).ToString();
+                data.fields[i].offset = (uint)(4 * i);
+                data.fields[i].datatype = 7; // FLOAT32
```

Comment:
use the declared types from pointfield message itself. https://github.com/RosProBridge/UnityBridge/blob/dcb365e693792e45bea06452ddcc931afa46cd57/Runtime/RosMsgs/Sensor/sensor_msgs.cs#L23